ts/Core/Templating.ts

```diff
@@ -416,7 +417,7 @@ function format(str = '', ctx: any, chart?: Chart): string {
 function numberFormat(
     number: number,
     decimals: number,
-    decimalPoint?: string,
+    decimalPoint?: string|Chart,
```

Comment:
`decimalPoint` being a Chart instance - sounds troublesome. How about we add another argument to the function or implement `numberFormat(number, { options })` support?

With a new argument/`options` object we would keep the backward compatibility as a bonus.

Reply:
At least we should rename the argument

Reply:
I see your point, will look into it. But back compat is already kept right? When the argument is a number.

Reply:
I took a look and found that we can use the chart as context instead, so we won't have to shoehorn in the chart instance as a parameter.